pkg/kubectl/describe.go

```diff
@@ -836,10 +836,29 @@ func describeNode(node *api.Node, pods []*api.Pod, events *api.EventList) (strin
 			fmt.Fprintf(out, "ExternalID:\t%s\n", node.Spec.ExternalID)
 		}
 		fmt.Fprintf(out, "Pods:\t(%d in total)\n", len(pods))
-		fmt.Fprint(out, "  Namespace\tName\n")
+		fmt.Fprint(out, "  Namespace\tName\tCPU-limit(milliCPU)\tMemory-limit(bytes)\n")
+		totalMilliCPU := int64(0)
+		totalMemory := int64(0)
+		fractionCPU := float64(0)
+		fractionMemory := float64(0)
 		for _, pod := range pods {
-			fmt.Fprintf(out, "  %s\t%s\n", pod.Namespace, pod.Name)
+			podTotalMilliCPU := int64(0)
+			podTotalMemory := int64(0)
+
+			for ix := range pod.Spec.Containers {
+				limits := pod.Spec.Containers[ix].Resources.Limits
+				podTotalMilliCPU += limits.Cpu().MilliValue()
+				podTotalMemory += limits.Memory().Value()
+			}
+			totalMilliCPU += podTotalMilliCPU
+			totalMemory += podTotalMemory
+			fmt.Fprintf(out, "  %s\t%s\t%d\t%d\n", pod.Namespace, pod.Name, podTotalMilliCPU, podTotalMemory)
```

Comment:
Is there a way to make sure these line up with the column headings? (In your sample output the last column (memory) didn't exactly line up.)